cmd/verify/verify.go

```diff
@@ -95,97 +95,97 @@ func Command() *cobra.Command {
 		&verifyConcurrency,
 		"concurrency",
 		0,
-		"number of tables to process at a time (defaults to number of CPUs)",
+		"Number of tables to process at a time (defaults to number of CPUs).",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyTableSplits,
 		"table-splits",
 		1,
-		"number of shards to break down each table into whilst doing row-based verification",
+		"Number of shards to break down each table into while doing row-based verification.",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyRowBatchSize,
 		"row-batch-size",
 		20000,
-		"number of rows to get from a table at a time",
+		"Number of rows to select from a table at a time.",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyLimitRowsPerSecond,
 		"rows-per-second",
 		0,
-		"if set, maximum number of rows to read per second during scanning per shard",
+		"If set, maximum number of rows to read per second during scanning per shard.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyFixup,
 		"fixup",
 		false,
-		"whether to fix up any rows",
+		"Whether to fix up any rows.",
 	)
 	cmd.PersistentFlags().DurationVar(
 		&verifyContinuousPause,
 		"continuous-pause-between-runs",
 		0,
-		"time to pause between continuous runs",
+		"Time to pause between continuous runs.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyContinuous,
 		"continuous",
 		false,
-		"whether verification should continuously run on each shard",
+		"Whether verification should continuously run on each shard.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyLive,
 		"live",
 		false,
-		"enables live mode, which attempts to account for rows that can change in value by retrying them before marking them as an inconsistency",
+		"Enables live mode, which attempts to account for rows that can change in value by retrying them before marking them as an inconsistency.",
 	)
 	cmd.PersistentFlags().BoolVar(
 		&verifyRows,
 		"rows",
 		true,
-		"whether rows should be verified (otherwise, performs a more basic schema check)",
+		"Whether rows should be verified (otherwise, performs a more basic schema check).",
 	)
 	cmd.PersistentFlags().IntVar(
 		&verifyLiveVerificationSettings.RunsPerSecond,
 		"live-runs-per-second",
 		verifyLiveVerificationSettings.RunsPerSecond,
-		"if set, sets a maximum number of attempts to reverify per second",
+		"If set, sets a maximum number of attempts to re-verify per second.",
```

Comment:
I assume this is live mode only.

```suggestion
		"Maximum number of row retry attempts per second (live mode only).",
```

Reply:
Good! Done.